Contest_API.md

```diff
@@ -513,9 +507,9 @@ The endpoints can be categorized into 4 groups as follows:
     awards, commentary;
   - Aggregate data: scoreboard, event-feed.
 
-Metadata is data about the API. These are not included in the event feed and
-are always required to be available. The access endpoint specifies which other
-endpoints are offered by the API. 
+Metadata is data about the API, and are the only required API endpoints. 
+They are not included in the event feed. The access endpoint specifies
+which other endpoints are offered by the API.
```

Comment:
I'd like to specify here or further down that anything specified in `access` must be provided by this API service. For example:
```suggestion
which other endpoints are offered by the API. That is, any endpoints and their properties listed in `access` must be provided (possibly with a `null` value when the property is optional), and only these endpoints and properties. 
```
I'd like this to be rigid in this sense, so that a client knows exactly what to expect, but throwing it out there for discussion.